src/components/app/details/appDetails/AppDetails.tsx

```diff
@@ -152,6 +153,7 @@ export const Details: React.FC<{
     const [detailedNode, setDetailedNode] = useState<{ name: string; containerName?: string }>(null);
     const [detailedStatus, toggleDetailedStatus] = useState<boolean>(false);
     const [commitInfo, showCommitInfo] = useState<boolean>(false)
+    const [urlInfo, showUrlInfo] = useState<boolean>(false)
```

Comment:
we should rename showUrlInfo to setUrlInfo

Reply:
done